app/controllers/works_controller.rb

```diff
@@ -152,6 +152,12 @@ def edit_button
 
   private
 
+  def globus_user_exists?(user_id)
+    return Settings.globus.test_user_exists if Settings.globus.test_mode && !Rails.env.production?
```

Comment:
Do you think we will want/need this ability when running tests? If not, I wonder whether we'd want `Rails.env.development?` or `!Rails.env.production?` here.

Reply:
no i guess we won't... i can do the switch to development ...  and then get rid of the test bits that force the state of this setting too

Reply:
OK, awesome!